### PR TITLE
Fix broken Gradle buildNative task

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java [hunk 1]
```diff
@@ -420,7 +420,7 @@ public void buildNative() {
             @Override
             public void accept(ConfigBuilder configBuilder) {
                 InMemoryConfigSource type = new InMemoryConfigSource(Integer.MAX_VALUE, "Native Image Type")
-                        .add("quarkus.package.types", "NATIVE");
+                        .add("quarkus.package.types", "native");
                 configBuilder.withSources(type);
```
Comment (gastaldi) [inline devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java]: You also need to add the `configs` below to the `configBuilder`

Comment (geoand) [reply devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java]: Yup, done

Comment (geoand) [reply devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java]: After the 10th try :P

Comment (gastaldi) [reply devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java]: Don't forget to remove this line then :)

Comment (geoand) [reply devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java]: :+1:

**Review by gastaldi — APPROVED** [review]
LGTM

**Review by gsmet — APPROVED** [review]
Changes make sense.

**Comment by gsmet** [comment]
@gastaldi did you check it fixes the issue for you too?

**Comment by gastaldi** [comment]
@geoand you also need to change https://github.com/quarkusio/quarkus/blob/master/devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusNative.java#L260 no?

**Comment by geoand** [comment]
Did you try it on the Gradle command line and see it not being applied?

**Comment by gastaldi** [comment]
I mean the option name, since the Mojo uses `auto-service-loader-registration` I figured the Gradle option should follow the same pattern (with a dash between `service` and `loader`)

**Comment by geoand** [comment]
Then yes it should. Feel free to do it since I need to take off for a while.

**Comment by gastaldi** [comment]
Cool, it's not a priority, so let's merge this PR first and I'll provide a PR later with this change

**Pull request merged**